README.md

```diff
@@ -176,25 +176,27 @@ The whole configuration is contained in file `config/{prod|dev|test}.exs` depend
 ### RESTful API configuration
 
 ```elixir
-config :maru, MongoosePush.Router,
-    versioning: [
-        using: :path
+config :mongoose_push, MongoosePushWeb.Endpoint,
```

Comment:
@rslota maybe we should consider changing it to be more meaningful as the previous version was, like `:phoenix`?

Reply:
It's not our call. Maru was configured as a separate app. Phoenix is a framework that injects the implementation to target project - all configuration is done via "our" app, and that's required by Phoenix, as Phoenix is the one that fetches this setting.